orchagent/intfsorch.cpp

```diff
@@ -78,10 +78,22 @@ sai_object_id_t IntfsOrch::getRouterIntfsId(const string &alias)
 {
     Port port;
     gPortsOrch->getPort(alias, port);
-    assert(port.m_rif_id);
     return port.m_rif_id;
 }
 
+string IntfsOrch::getRouterIntfsAlias(const IpAddress &ip)
```

Comment:
This function is ambiguous as same IP can be existing on multiple interfaces in a multi VRF scenario. I don't think we can have a function like this. Is it for some backward compatibility?

Reply:
This function will add a vrf argument to support VRF. The VRF PR includes this. Mirror session is using it now, in getNeighborEntry.

Reply:
Can you add VRF argument to this function in this PR? It can pass `default vrf` from the caller.

Reply:
OK, actually I had removed it just for this PR.